src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseFileController.cs

```diff
@@ -70,31 +70,31 @@ public async Task<ActionResult> Stream(Guid releaseId, Guid id)
         }
 
         [HttpGet("release/{releaseId}/ancillary")]
-        public async Task<ActionResult<IEnumerable<FileInfo>>> GetAncillaryFiles(Guid releaseId)
+        public async Task<ActionResult<IEnumerable<AncillaryFileInfo>>> GetReleaseAncillaryFilesInfo(Guid releaseId)
```

Comment:
I think I would have just kept this as GetAncillaryFiles throughout the controller, service and tests. Seems a bit verbose.

Reply:
I'll change it before merging